chain/substreams/src/data_source.rs

```diff
@@ -340,14 +346,35 @@ mod test {
         );
     }
 
+    fn gen_package() -> graph::substreams::Package {
+        graph::substreams::Package {
+            proto_files: vec![],
+            version: 0,
+            modules: Some(graph::substreams::Modules {
+                modules: vec![graph::substreams::Module {
+                    name: "output".into(),
+                    initial_block: 123,
+                    binary_entrypoint: "entry".into(),
+                    binary_index: 0,
+                    kind: None,
+                    inputs: vec![],
+                    output: None,
+                }],
```

Comment:
Actually, could you add a second module with a different block so we see the code is working properly.